nemo/collections/asr/parts/submodules/rnnt_beam_decoding.py

```diff
@@ -1024,16 +1036,18 @@ def modified_adaptive_expansion_search(
                             list_b.append(new_hyp)
                         else:
                             # If the expansion was a token
-                            new_hyp.y_sequence.append(int(k))
-
-                            # TODO: Setup LM
-                            if self.language_model is not None:
-                                # new_hyp.score += self.lm_weight * float(
-                                #     hyp.lm_scores[k]
-                                # )
-                                pass
-
-                            list_exp.append(new_hyp)
+                            # new_hyp.y_sequence.append(int(k))
```

Comment:
Should we drop this line?

Reply:
Ah i kinda kept it cause i might need it for debugging during rnnt beam setup